cider-velox/src/substrait/VeloxPlanFragmentToSubstraitPlanOptions.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include "VeloxPlanFragmentToSubstraitPlanOptions.h"
+
+DEFINE_string(substrait_yaml_root_path, "", "Substrait extension root path");
```

Comment:
in which case, we should add this path? Let's add some example in project website. We can add it under [section](https://intel.github.io/BDTK/user.html).

Also let's combine this with other options. ```cider-velox/src/ciderTransformer/CiderPlanTransformerOptions.cpp```. How about calling it ```veloxCiderOption``` and put it under cider-velox/src/option path. We can do it in follow-up PR once https://github.com/intel/BDTK/pull/179/files merged.